tools/funclatency.py

```diff
@@ -12,11 +12,6 @@
 # The pattern is a string with optional '*' wildcards, similar to file
 # globbing. If you'd prefer to use regular expressions, use the -r option.
 #
-# Currently nested or recursive functions are not supported properly, and
-# timestamps will be overwritten, creating dubious output. Try to match single
-# functions, or groups of functions that run at the same stack layer, and
-# don't ultimately call each other.
-#
 # Copyright (c) 2015 Brendan Gregg.
```

Comment:
This is rather a big change. You can add a "Copyright" note for your contribution here if you want. Thanks!